src/AppEndToEnd.test.tsx

```diff
@@ -170,7 +173,14 @@ it('VxMark+Print end-to-end flow', async () => {
     // Vote for candidate contest
     if (title === presidentContest.title) {
       fireEvent.click(getByText(presidentContest.candidates[0].name))
+
+      // advance time by a second, run timers, and see if things get saved
+      advanceBy(1100)
```

Comment:
The comment is out of sync with the code. Replace "advance time by a second" with the reason we need to wait, i.e. what are we waiting for?

Reply:
I don't think it's out of sync, but I'll clarify the comment.